src/Resources.Designer.cs

```diff
@@ -0,0 +1,638 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
```

Comment:
how is this generated ?
do we need to add it inside the repo ? (or can the build generate it on the fly)
how to we _property_ add new error codes in it ? (should be added to the README.md)

Reply:
It's generated by visual studio when there are updates to `Resources.resx`.  For now, I am including it in the repo because that's what the android folks are doing however I will investigate it further.